### ISSUE TITLE
Application fonts are not loading

### ISSUE THREAD

**Issue by SHSharkar** [laravel-shift/workbench-issues#52]
**Desktop:**
 - OS: Microsoft Windows 10 Pro
 - OS Version: 10.0.19044 Build 19044
 - App Version: 0.7.2.0

**Describe the bug**
Today I have reinstalled the operating system and set up the Shift Workbench. After installation is complete, the font for the interface is broken.

**To Reproduce**
1. Install the latest version of Shift Workbench on Windows 10
2. Open the application

![NBaAas5r4z](https://user-images.githubusercontent.com/12555542/149947164-8acaea92-2583-4d5b-b8a2-e865995946b3.png)
![SKz5EBN82A](https://user-images.githubusercontent.com/12555542/149947293-64ad97bf-e7f3-43e3-ac3d-fadb3f6bdb03.png)

**Comment by jasonmccreary** [comment]
Given that you reinstalled the OS, I'm inclined to say this is system related. I don't have access to a Windows 10 machine. So unless someone else reports this same issue, I'm not sure how best to proceed.

**Comment by SHSharkar** [comment]
Hello, and thanks for your response. I have tried to figure out the issue to resolve.

It seems I have recently installed the `Open Sans Condensed` font family to my operating system.

I deleted the fonts and restarted the operating system after doing the changes. It seems the problem resolves.

Therefore, it was due to the FONT issue. I am going to close the issue.

I hope someone will find this issue helpful soon if they do a similar change.

![LJCHrWt4Oo](https://user-images.githubusercontent.com/12555542/150006610-d0b55f04-b9dc-4446-b4ee-39a02b9128a7.png)
